generator/generator.go

```diff
@@ -189,6 +189,7 @@ func (g *Generator) Generate(f *ast.File) ([]byte, error) {
 		// Parse the enum doc statement
 		enum, pErr := g.parseEnum(ts)
 		if pErr != nil {
+			fmt.Println(pErr)
```

Comment:
Please use Sentinel errors to determine if we should print the error or not. There are other errors in there that do not need to be printed, and would just cause a bunch of noise.